core/src/commonMain/kotlin/com/sourcepoint/mobile_core/network/requests/MessagesRequest.kt

```diff
@@ -60,6 +61,13 @@ data class MessagesRequest(
                 val hasLocalData: Boolean,
                 val status: CCPAConsent.CCPAConsentStatus?
             )
+
+            @Serializable
+            data class Preferences(
+                val targetingParams: SPTargetingParams?,
+                val hasLocalData: Boolean,
+                val consentStatus: ConsentStatus?
```

Comment:
Are you sure it should be `ConsentStatus` here?

Reply:
I've answered this in the first comment of the review